### DISCUSSION TITLE
Does the SvelteKit User Management example has to be this complex ?

### DISCUSSION THREAD

**Discussion by hocman2** [supabase/supabase#26717] [Questions]
Simply setting up the barebones requires a ton of different nested layouts and a simple request to the home page triggers 5 warnings: 
"Using the user object as returned from supabase.auth.getSession() or from some supabase.auth.onAuthStateChange() events could be insecure! [...]"
Making the console completely unreadable

Isn't there a less convoluted way to have roughly the same setup ?

**Comment by j4w8n** [comment]
The app looks pretty standard to me. There's only one layout. You might be able to simplify it by not using the SSR package, and do auth completely client-side, but most people seem to prefer server-side auth so that's what the docs use.

The warnings are a pain, and there have been issues created. It's especially troublesome for SvelteKit. https://github.com/supabase/auth-js/issues/888
